lib/mustache/settings.rb

```diff
@@ -2,6 +2,30 @@
 # view class, or a single Mustache instance.
 class Mustache
 
+  def initialize_settings
+    @template = nil
+    @template_path = nil
+    @template_extension = nil
+    @template_name = nil
+    @template_file = nil
+    @raise_on_context_miss = nil
+  end
+
+  def self.initialize_settings
+    @template = nil
+    @template_path = nil
+    @template_extension = nil
+    @template_name = nil
+    @template_file = nil
+    @raise_on_context_miss = nil
+  end
```

Comment:
I'm personally not sure if this is the right way to do it, but I'm not sure if there is a better way.

Reply:
It does seem kind of odd having the class method setting these

Reply:
I agree, but the design of this library is class-centric, so these settings are per-class not per-instance. So, the only way to change this would be to some how change the design of the library.